.github/workflows/publish_rondb.yml

```diff
@@ -0,0 +1,71 @@
+name: Publish RonDB Tarball
+on:
+  push:
+    branches:
+      - "21.04.[0-9]+"
+      - "22.10.[0-9]+"
+      - "24.10.[0-9]+"
+
+# This is important because the jobs are expensive and we don't want to wait a lot of time.
+# This cancel-in-progress only works per branch,  if a previous commit launched a job, a
+# new one will cancel it.
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref_name }}
+  cancel-in-progress: true
+
+jobs:
+  publish-arm-tarball:
+    runs-on: [self-hosted, ARM64]
+    if: github.repository == 'logicalclocks/rondb'
+    env:
+      REPO_HOPSWORKS_HOSTNAME: repo.hops.works
+      REPO_HOPSWORKS_USER: repo_upload
+      SSH_PRIVATE_KEY: ${{ secrets.REPO_UPLOAD_SSH_PRIVATE_KEY }}
+
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
+
+      - name: Set up SSH
+        run: |
+          mkdir -p ~/.ssh
+          echo "$SSH_PRIVATE_KEY" > ~/.ssh/id_rsa
+          chmod 600 ~/.ssh/id_rsa
+          ssh-keyscan -H $REPO_HOPSWORKS_HOSTNAME >> ~/.ssh/known_hosts
```

Comment:
Can we get this from any other source? E.g. hard-code it like the other vars above?